components/Header/Header.js

```diff
@@ -0,0 +1,77 @@
+import styled from "styled-components";
+import Image from "next/image";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import dynamic from "next/dynamic";
+
+const StyledHeader = styled.header`
+  text-align: center;
+  margin-bottom: 0rem;
```

Comment:
You don't need a unit when the value is '0': 
```suggestion
  margin-bottom: 0;
```